code/workspaces/web-app/src/components/stacks/StackCardActions.js

```diff
@@ -57,12 +57,12 @@ export const PureStackCardActions = ({ stack, openStack, deleteStack, editStack,
           Open
         </PrimaryActionButton>
       </PermissionWrapper>}
-      {ownsStack && <PermissionWrapper className={classes.buttonWrapper} userPermissions={userPermissions} permission={deletePermission}>
+      {ownsStack && stack.status && <PermissionWrapper className={classes.buttonWrapper} userPermissions={userPermissions} permission={deletePermission}>
```

Comment:
How come there is a check for `stack.status` here?

Reply:
It's a bit hacky, but without this the menu appears on the "Preview Card" when creating storage which looks really odd, so just checking against something that applies to a 'real' stack